### PR TITLE
Meetings plugin widget

### PR DESCRIPTION
**Pull request by mwegman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,30 +56,32 @@
...
+    "@webex/internal-plugin-presence": "1.98.2",
+    "@webex/internal-plugin-search": "1.98.2",
+    "@webex/internal-plugin-team": "1.98.2",
+    "@webex/internal-plugin-user": "1.98.2",
+    "@webex/internal-plugin-wdm": "1.98.2",
+    "@webex/plugin-authorization": "1.98.2",
+    "@webex/plugin-logger": "1.98.2",
+    "@webex/plugin-meetings": "1.98.2",
+    "@webex/plugin-people": "1.98.2",
+    "@webex/plugin-phone": "1.98.2",
```
Comment (adamweeks) [inline package.json:79]: Can we remove this now? We can also remove the import [here](https://github.com/webex/react-widgets/blob/master/packages/node_modules/@webex/react-redux-spark/src/sdk.js#L6)

Comment (mwegman) [reply package.json:79]: I will give it a shot and see if it breaks anything.

File: packages/node_modules/@webex/widget-space/events.md [hunk 2]
```diff
@@ -73,109 +73,6 @@ A previously unread room is now fully read.
 }
 ```
 
-## Calls
```
Comment (adamweeks) [inline packages/node_modules/@webex/widget-space/events.md:76]: We need to remove this section from the recents events docs too?

File: packages/node_modules/@webex/widget-recents/src/selector.js [hunk 3]
```diff
@@ -27,6 +29,17 @@ const getMercuryStatus = (state) => state.mercury.get('status');
 const getFeatures = (state) => state.features;
 const getWidgetProps = (state, props) => props;
 
+const getSDKAdapter = createSelector(
+  [getSpark],
```
Comment (adamweeks) [inline packages/node_modules/@webex/widget-recents/src/selector.js:33]: Does this create a new sdk adapter every time the spark module changes?

File: packages/node_modules/@webex/redux-module-media/src/actions.js [hunk 4]
```diff
@@ -217,67 +244,66 @@ export function listenForCalls(sparkInstance) {
...
+    .then(tap((meeting) => processCall({dispatch, call: {...meeting}, destination})))
+    .then((meeting) => {
+      const bye = meeting.leave;
+
+      // eslint-disable-next-line no-param-reassign
+      meeting.leave = (...args) => bye.bind(meeting)(...args)
+        .then(() => dispatch(removeCall(meeting.id)))
+        .then(() => dispatch(cleanUp()))
+        // eslint-disable-next-line no-param-reassign
+        .finally(() => { meeting.leave = bye.bind(meeting); });
```
Comment (taymoork2) [inline packages/node_modules/@webex/redux-module-media/src/actions.js:302]: Does this work?

```suggestion
        .finally(() => (meeting.leave = bye.bind(meeting)));
```

File: packages/node_modules/@webex/widget-recents/src/selector.js (hunk 3 above)
Comment (mwegman) [reply packages/node_modules/@webex/widget-recents/src/selector.js:33]: This creates a new SDK adapter every time there is a new spark module instantiated.

File: packages/node_modules/@webex/redux-module-media/src/actions.js (hunk 4 above)
Comment (mwegman) [reply packages/node_modules/@webex/redux-module-media/src/actions.js:302]: Yes it is the same as:
```suggestion
        .finally(() => {
          meeting.leave = bye.bind(meeting);
        });
```
It's a function that does one assignment.

**Comment by taymoork2** [comment]
It's failing to build (not npm i) on AWS
```
Module build failed (from /codebuild/output/src299385700/src/react-widgets/node_modules/sass-loader/dist/cjs.js):
@import '~@momentum-ui/core/scss/momentum-ui';
                                 ^
File to import not found or unreadable: ~@momentum-ui/core/scss/momentum-ui.
```

**Review by taymoork2 — APPROVED** [review]
LGTM

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Awesome work @mwegman!

I just noticed minor spelling inconsistencies with "Adapter" and "Adaptor", the correct spelling is "adapter" but not blocking.
I did noticed that when I had an incoming call, and then I hung up, the timer still kept going
meeting-timer

File: packages/node_modules/@webex/react-redux-spark/src/actions.js [hunk 5]
```diff
@@ -43,6 +44,12 @@ export function storeSparkInstance(spark) {
   };
 }
 
+export const storeSparkAdaptor = (adaptor) => ({
```
[the remark this answers is not in the document]
Comment (mwegman) [reply packages/node_modules/@webex/react-redux-spark/src/actions.js:47]: https://grammarist.com/spelling/adapter-adaptor/

File: package.json [hunk 6]
```diff
@@ -142,10 +144,10 @@
     "@rollup/plugin-url": "^4.0.1",
     "@webex/eslint-config-base": "^0.8.0",
     "@webex/eslint-config-react": "^0.8.0",
-    "@webex/test-helper-appid": "1.80.271",
-    "@webex/test-helper-file": "1.80.271",
-    "@webex/test-helper-server": "1.80.271",
-    "@webex/test-helper-test-users": "1.80.271",
+    "@webex/test-helper-appid": "1.98.2",
```
[the remark this answers is not in the document]
Comment (mwegman) [reply package.json:147]: I updated them to the same version as the main plugins but tests started to fail. I figured we could create a JIRA to update the test packages at a later date.

File: packages/node_modules/@webex/react-redux-spark/src/actions.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (lalli-flores) [reply packages/node_modules/@webex/react-redux-spark/src/actions.js:47]: All I'm saying is that is not consistent, some files have "adapter" and some "adaptor"

**Review by lalli-flores — APPROVED** [review]
Issues fixed, thanks!

**Pull request merged**